Airports/EGKK/Ownership.txt

```diff
@@ -26,7 +27,7 @@ ARRAPT:EGKK
 SECTOR:KKFIN_E:0:4500
 OWNER:KKF:KKA:TCSW:TCS:TC:LS:LWS:LSC:L
 ALTOWNER:Observing LON_S_CTR:KKF:KKA:TCSW:TCS:TC:O:LM:LS:LWS:LSC:L
-ALTOWNER:Observing LTC_S_CTR:KKF:KKA:TCSW:O:LM:TCS:TC:LS:LWS:LSC:L
+ALTOWNER:Observing LTC_S_CTR:KKF:KKA:TCSW:TCS:TC:LS:LWS:O:LM:LSC:L
```

Comment:
You've edited the wrong thing here! Need to add a line for your SC, and revert this line back to what it was